operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -29,6 +29,7 @@ kiali_defaults:
     #additional_service_yaml:
     image_name: "kiali/kiali"
     image_pull_policy: "IfNotPresent"
+    image_pull_secrets: []
```

Comment:
The default value will be always an empty array ?

Reply:
yes. by default you have no secrets define (and none are required to pull from docker.io or quay.io). You can optionally set this to one or more secret names if you need to pull images from a private image registry.